### ISSUE TITLE
fix: shorebird patch iOS `Failed to link AOT files: Exception: Failed to link:`

### ISSUE THREAD

**Issue by lachlanjcollins** [shorebirdtech/shorebird#2480]
**Description**

After successfully creating an iOS release I am unable to create a patch. I get the following error message:
`Failed to link AOT files: Exception: Failed to link:`

**Steps To Reproduce**

1. Create an iOS release
```
shorebird release ios \
    --flavor dev \
    --export-options-plist $EXPORT_OPTIONS_PLIST_PATH \
    -- --split-debug-info=$DEBUG_SYMBOLS_IOS_PATH
```
2. Apply a patch
```
shorebird patch ios \
    --flavor dev --verbose \
    --export-options-plist $EXPORT_OPTIONS_PLIST_PATH \
    -- --split-debug-info=$DEBUG_SYMBOLS_IOS_PATH
```

**Expected Behavior**

The patch is successfully uploaded.

**Screenshots**
![Screenshot 2024-09-10 at 3 34 15 PM](https://github.com/user-attachments/assets/123b4d2e-726f-4a3e-a3dc-02e5224d6255)

**Additional Context**
- Using flutter version `3.24.2`
- Would be happy to jump in a Discord call to run through our process / show the shorebird log file if that would assist, thanks :)

**Comment by eseidel** [comment]
Thank you very much for the report!

**Comment by felangel** [comment]
@lachlanjcollins thanks for the report! I’d love to hop on a call and debug the issue. Feel free to ping me on discord whenever works best for you, thanks!

**Comment by maroafenogho** [comment]
Is there any update on this?
I am experiencing the same thing.

**Comment by felangel** [comment]
@maroafenogho are you able to re-run the command using the `--verbose` flag and share the output as well as provide any reproduction steps? We're more than happy to help! You can also reach out to us directly on [discord](https://discord.gg/shorebird).

**Comment by zeucxb** [comment]
We are facing exactly the same problem here, adding `verbose` don't give any additional information or change the error message.

**Comment by zeucxb** [comment]
I've also tried `shorebird cache clean`

**Comment by lachlanjcollins** [comment]
Hey @felangel just confirming I've reached out on discord 😊

**Comment by maroafenogho** [comment]
@felangel 
I have reached out on discord. Please confirm that you have seen it. Thanks

**Comment by eseidel** [comment]
We're aware, yes.  Felix is on it, thanks.

**Comment by felangel** [comment]
@zeucxb @lachlanjcollins @maroafenogho we just released v1.3.4 of the Shorebird CLI which includes improved logging when the linker fails. Are you able to upgrade to 1.3.4, retry patching, and share the full output? Thanks!

**Comment by maroafenogho** [comment]
> @zeucxb @lachlanjcollins @maroafenogho we just released v1.3.4 of the Shorebird CLI which includes improved logging when the linker fails. Are you able to upgrade to 1.3.4, retry patching, and share the full output? Thanks!


```
Failed to link AOT files: Exception: Failed to link:
stdout: [Process] /Users/maro/.shorebird/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --dump_class_table_link_data=/var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/dyYG7K/Products/Applications/Runner.app/Frameworks/App.framework/App.ct.link --dump_dispatch_table_link_data=/var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/dyYG7K/Products/Applications/Runner.app/Frameworks/App.framework/App.dt.link --dump_object_pool_link_data=/var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/dyYG7K/Products/Applications/Runner.app/Frameworks/App.framework/App.op.link /var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/dyYG7K/Products/Applications/Runner.app/Frameworks/App.framework/App
[Process] /Users/maro/.shorebird/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --dump_class_table_link_data=/Users/maro/Documents/flutter apps/ologo-mobile/build/out.ct.link /Users/maro/Documents/flutter apps/ologo-mobile/build/out.aot
[Process] /Users/maro/.shorebird/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --shorebird --out=/var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/7xF5Zc/App.analyze_snapshot.json /var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/dyYG7K/Products/Applications/Runner.app/Frameworks/App.framework/App
[Process] /Users/maro/.shorebird/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --dump_pool /var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/dyYG7K/Products/Applications/Runner.app/Frameworks/App.framework/App
[Process] /Users/maro/.shorebird/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --dump_class_table --out=/var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/7xF5Zc/App.class_table.json /var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/dyYG7K/Products/Applications/Runner.app/Frameworks/App.framework/App
[Process] /Users/maro/.shorebird/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --shorebird --out=/var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/7xF5Zc/out.analyze_snapshot.json /Users/maro/Documents/flutter apps/ologo-mobile/build/out.aot
[Process] /Users/maro/.shorebird/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --dump_pool /Users/maro/Documents/flutter apps/ologo-mobile/build/out.aot
[Process] /Users/maro/.shorebird/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --dump_class_table --out=/var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/7xF5Zc/out.class_table.json /Users/maro/Documents/flutter apps/ologo-mobile/build/out.aot
[Process] /Users/maro/.shorebird/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/gen_snapshot_arm64 --snapshot_kind=app-aot-elf --base_ct_link_data=/var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/dyYG7K/Products/Applications/Runner.app/Frameworks/App.framework/App.ct.link --patch_ct_link_data=/Users/maro/Documents/flutter apps/ologo-mobile/build/out.ct.link --print_dispatch_table_to=/var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/7xF5Zc/out.ct.dispatch_table.json --elf=/Users/maro/Documents/flutter apps/ologo-mobile/build/out.ct.aot /Users/maro/Documents/flutter apps/ologo-mobile/build/app.dill
[Process] /Users/maro/.shorebird/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --shorebird --out=/var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/7xF5Zc/out.ct.analyze_snapshot.json /Users/maro/Documents/flutter apps/ologo-mobile/build/out.ct.aot
[Process] /Users/maro/.shorebird/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --dump_pool /Users/maro/Documents/flutter apps/ologo-mobile/build/out.ct.aot
[Process] /Users/maro/.shorebird/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --dump_class_table --out=/var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/7xF5Zc/out.ct.class_table.json /Users/maro/Documents/flutter apps/ologo-mobile/build/out.ct.aot
[Process] /Users/maro/.shorebird/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --dump_object_pool_link_data=/Users/maro/Documents/flutter apps/ologo-mobile/build/out.op.link /Users/maro/Documents/flutter apps/ologo-mobile/build/out.ct.aot
[Process] /Users/maro/.shorebird/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/gen_snapshot_arm64 --snapshot_kind=app-aot-elf --base_ct_link_data=/var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/dyYG7K/Products/Applications/Runner.app/Frameworks/App.framework/App.ct.link --patch_ct_link_data=/Users/maro/Documents/flutter apps/ologo-mobile/build/out.ct.link --base_op_link_data=/var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/dyYG7K/Products/Applications/Runner.app/Frameworks/App.framework/App.op.link --patch_op_link_data=/Users/maro/Documents/flutter apps/ologo-mobile/build/out.op.link --base_dt_link_data=/var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/dyYG7K/Products/Applications/Runner.app/Frameworks/App.framework/App.dt.link --print_dispatch_table_to=/var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/7xF5Zc/out.optimized.dispatch_table.json --elf=/Users/maro/Documents/flutter apps/ologo-mobile/build/out.optimized.aot /Users/maro/Documents/flutter apps/ologo-mobile/build/app.dill
[Process] /Users/maro/.shorebird/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --shorebird --out=/var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/7xF5Zc/out.optimized.analyze_snapshot.json /Users/maro/Documents/flutter apps/ologo-mobile/build/out.optimized.aot
[Process] /Users/maro/.shorebird/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --dump_pool /Users/maro/Documents/flutter apps/ologo-mobile/build/out.optimized.aot
[Process] /Users/maro/.shorebird/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --dump_class_table --out=/var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/7xF5Zc/out.optimized.class_table.json /Users/maro/Documents/flutter apps/ologo-mobile/build/out.optimized.aot
[Process] /Users/maro/.shorebird/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --shorebird --out=/var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/dyYG7K/Products/Applications/Runner.app/Frameworks/App.framework/App.analyze_snapshot.json /var/folders/jv/08k84vkn0sl8q9jy2_jwf05w0000gn/T/dyYG7K/Products/Applications/Runner.app/Frameworks/App.framework/App
[Process] /Users/maro/.shorebird/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --shorebird --out=/Users/maro/Documents/flutter apps/ologo-mobile/build/out.optimized.analyze_snapshot.json /Users/maro/Documents/flutter apps/ologo-mobile/build/out.optimized.aot

stderr:  (141.6s)
```

**Comment by dfdgsdfg** [comment]
Same here.

```
✗ Failed to link AOT files: Exception: Failed to link:
stdout: [Process] /Users/dididi/.local/share/mise/installs/shorebird/1.3.4/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --dump_class_table_link_data=/var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/xH83fz/Products/Applications/Runner.app/Frameworks/App.framework/App.ct.link --dump_dispatch_table_link_data=/var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/xH83fz/Products/Applications/Runner.app/Frameworks/App.framework/App.dt.link --dump_object_pool_link_data=/var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/xH83fz/Products/Applications/Runner.app/Frameworks/App.framework/App.op.link /var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/xH83fz/Products/Applications/Runner.app/Frameworks/App.framework/App
[Process] /Users/dididi/.local/share/mise/installs/shorebird/1.3.4/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --dump_class_table_link_data=/Users/dididi/workspaces/us/us-app-prod/apps/insight_app/build/out.ct.link /Users/dididi/workspaces/us/us-app-prod/apps/insight_app/build/out.aot
[Process] /Users/dididi/.local/share/mise/installs/shorebird/1.3.4/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --shorebird --out=/var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/0hRlGA/App.analyze_snapshot.json /var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/xH83fz/Products/Applications/Runner.app/Frameworks/App.framework/App
[Process] /Users/dididi/.local/share/mise/installs/shorebird/1.3.4/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --dump_pool /var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/xH83fz/Products/Applications/Runner.app/Frameworks/App.framework/App
[Process] /Users/dididi/.local/share/mise/installs/shorebird/1.3.4/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --dump_class_table --out=/var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/0hRlGA/App.class_table.json /var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/xH83fz/Products/Applications/Runner.app/Frameworks/App.framework/App
[Process] /Users/dididi/.local/share/mise/installs/shorebird/1.3.4/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --shorebird --out=/var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/0hRlGA/out.analyze_snapshot.json /Users/dididi/workspaces/us/us-app-prod/apps/insight_app/build/out.aot
[Process] /Users/dididi/.local/share/mise/installs/shorebird/1.3.4/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --dump_pool /Users/dididi/workspaces/us/us-app-prod/apps/insight_app/build/out.aot
[Process] /Users/dididi/.local/share/mise/installs/shorebird/1.3.4/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --dump_class_table --out=/var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/0hRlGA/out.class_table.json /Users/dididi/workspaces/us/us-app-prod/apps/insight_app/build/out.aot
[Process] /Users/dididi/.local/share/mise/installs/shorebird/1.3.4/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/gen_snapshot_arm64 --snapshot_kind=app-aot-elf --base_ct_link_data=/var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/xH83fz/Products/Applications/Runner.app/Frameworks/App.framework/App.ct.link --patch_ct_link_data=/Users/dididi/workspaces/us/us-app-prod/apps/insight_app/build/out.ct.link --print_dispatch_table_to=/var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/0hRlGA/out.ct.dispatch_table.json --elf=/Users/dididi/workspaces/us/us-app-prod/apps/insight_app/build/out.ct.aot /Users/dididi/workspaces/us/us-app-prod/apps/insight_app/build/app.dill
[Process] /Users/dididi/.local/share/mise/installs/shorebird/1.3.4/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --shorebird --out=/var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/0hRlGA/out.ct.analyze_snapshot.json /Users/dididi/workspaces/us/us-app-prod/apps/insight_app/build/out.ct.aot
[Process] /Users/dididi/.local/share/mise/installs/shorebird/1.3.4/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --dump_pool /Users/dididi/workspaces/us/us-app-prod/apps/insight_app/build/out.ct.aot
[Process] /Users/dididi/.local/share/mise/installs/shorebird/1.3.4/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --dump_class_table --out=/var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/0hRlGA/out.ct.class_table.json /Users/dididi/workspaces/us/us-app-prod/apps/insight_app/build/out.ct.aot
[Process] /Users/dididi/.local/share/mise/installs/shorebird/1.3.4/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --dump_object_pool_link_data=/Users/dididi/workspaces/us/us-app-prod/apps/insight_app/build/out.op.link /Users/dididi/workspaces/us/us-app-prod/apps/insight_app/build/out.ct.aot
[Process] /Users/dididi/.local/share/mise/installs/shorebird/1.3.4/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/gen_snapshot_arm64 --snapshot_kind=app-aot-elf --base_ct_link_data=/var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/xH83fz/Products/Applications/Runner.app/Frameworks/App.framework/App.ct.link --patch_ct_link_data=/Users/dididi/workspaces/us/us-app-prod/apps/insight_app/build/out.ct.link --base_op_link_data=/var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/xH83fz/Products/Applications/Runner.app/Frameworks/App.framework/App.op.link --patch_op_link_data=/Users/dididi/workspaces/us/us-app-prod/apps/insight_app/build/out.op.link --base_dt_link_data=/var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/xH83fz/Products/Applications/Runner.app/Frameworks/App.framework/App.dt.link --print_dispatch_table_to=/var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/0hRlGA/out.optimized.dispatch_table.json --elf=/Users/dididi/workspaces/us/us-app-prod/apps/insight_app/build/out.optimized.aot /Users/dididi/workspaces/us/us-app-prod/apps/insight_app/build/app.dill
[Process] /Users/dididi/.local/share/mise/installs/shorebird/1.3.4/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --shorebird --out=/var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/0hRlGA/out.optimized.analyze_snapshot.json /Users/dididi/workspaces/us/us-app-prod/apps/insight_app/build/out.optimized.aot
[Process] /Users/dididi/.local/share/mise/installs/shorebird/1.3.4/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --dump_pool /Users/dididi/workspaces/us/us-app-prod/apps/insight_app/build/out.optimized.aot
[Process] /Users/dididi/.local/share/mise/installs/shorebird/1.3.4/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --dump_class_table --out=/var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/0hRlGA/out.optimized.class_table.json /Users/dididi/workspaces/us/us-app-prod/apps/insight_app/build/out.optimized.aot
[Process] /Users/dididi/.local/share/mise/installs/shorebird/1.3.4/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --shorebird --out=/var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/xH83fz/Products/Applications/Runner.app/Frameworks/App.framework/App.analyze_snapshot.json /var/folders/56/tv5k0s792ys4ck095k3rtkq40000gn/T/xH83fz/Products/Applications/Runner.app/Frameworks/App.framework/App
[Process] /Users/dididi/.local/share/mise/installs/shorebird/1.3.4/bin/cache/flutter/6e83d825c469beb6e98e20c5ec69382f7f554fe7/bin/cache/artifacts/engine/ios-release/analyze_snapshot_arm64 --shorebird --out=/Users/dididi/workspaces/us/us-app-prod/apps/insight_app/build/out.optimized.analyze_snapshot.json /Users/dididi/workspaces/us/us-app-prod/apps/insight_app/build/out.optimized.aot
```


```
shorebird doctor
Shorebird 1.3.4 • git@github.com:shorebirdtech/shorebird.git
Flutter 3.24.3 • revision 6e83d825c469beb6e98e20c5ec69382f7f554fe7
Engine • revision 9e5366d03255b882152444ab8bb8078afcb6fd54

✗ Shorebird is up-to-date (5.9s)
  [✗] Failed to get shorebird version. Error: fatal: HEAD가 브랜치를 가리키지 않습니다
✓ Flutter install is correct (0.5s)
✓ AndroidManifest.xml files contain INTERNET permission (36ms)
✓ Has access to api.shorebird.dev (0.5s)
✓ Has access to storage.googleapis.com (0.3s)
✓ shorebird.yaml found in pubspec.yaml assets (17ms)
1 issue detected.
```

**Comment by felangel** [comment]
Can you manually run the last analyze_snapshot command from the root of your project and paste the output? Thanks!

**Comment by felangel** [comment]
Shorebird v1.3.5 now contains several fixes to improve the logging. Can everyone who is still facing this issue please upgrade to v1.3.5, re-release, and re-run `shorebird patch ios --verbose` and share the logs? Thanks!

**Comment by eseidel** [comment]
One customer reported the failure was "base and patch snapshots have differing VM sections".

**Comment by eseidel** [comment]
Felix is still looking to learn more here.  We would love to sit down and debug with someone experiencing this issue.  https://discord.gg/shorebird reaches us or https://calendly.com/eseidel for scheduling a call.

One possible cause is passing `--obfuscate` during release, but not during patch (or vice versa).

**Comment by maroafenogho** [comment]
> One customer reported the failure was "base and patch snapshots have differing VM sections".



> Felix is still looking to learn more here. We would love to sit down and debug with someone experiencing this issue. https://discord.gg/shorebird reaches us or https://calendly.com/eseidel for scheduling a call.
> 
> One possible cause is passing `--obfuscate` during release, but not during patch (or vice versa).

I feel this may be what led to the issues we faced. I will give feedback when I am able to.
Thank you.

**Comment by eseidel** [comment]
Pablo kindly hopped on a call with Felix and I this morning and Felix and he were able to figure out that the flag that's causing the trouble (for his patch) is --split-debug-info.  Removing that flag (we recognize that's not a great option) in both release and patch builds fixed the issue.  We're not yet sure why that's breaking Shorebird patch, but Felix is investigating now.

**Comment by eseidel** [comment]
I'm embarrassed to realize that @lachlanjcollins had already explained that in the original filing and clearly I hadn't read closely enough.  (Too much support mail to keep up with.) Regardless, Felix is investigating now.

**Comment by felangel** [comment]
This should be fixed in v1.4.5 of Shorebird CLI 🎉
